umpf

```diff
@@ -1011,6 +1011,11 @@ rebase_end() {
 		sed -i -r "s:^(EXTRAVERSION =.*)\$:\1-${version}:" Makefile
 		git add Makefile
 	fi
+	if grep -sq '^VERSION_STRING\s*:=' Makefile; then
```

Comment:
I think we should make sure that only on place is patched in the Makefile:
```suggestion
	elif grep -sq '^VERSION_STRING\s*:=' Makefile; then
```

Reply:
Yes you're right. I will update this accordingly.